bundles/org.openhab.binding.lgwebos/pom.xml

```diff
@@ -15,21 +15,9 @@
 
   <dependencies>
     <dependency>
-      <groupId>xpp3</groupId>
-      <artifactId>xpp3</artifactId>
-      <version>1.1.4c</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.json</groupId>
-      <artifactId>json</artifactId>
-      <version>20140107</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.openhab.osgiify</groupId>
-      <artifactId>org.java-websocket.java-websocket</artifactId>
-      <version>1.3.2</version>
+      <groupId>org.eclipse.jetty.websocket</groupId>
+      <artifactId>websocket-client</artifactId>
```

Comment:
Do you need to websocket dependency? I would expect it be available through the parent pom?

Reply:
Indeed, it compiles without it. Taking it out.